test/models/content_view_test.rb

```diff
@@ -212,6 +212,40 @@ def test_all_version_library_instances_not_empty
       refute_empty @library_view.all_version_library_instances
     end
 
+    test_attributes :pid => '4a3b616d-53ab-4396-9a50-916d6c42a401'
+    def test_should_create_composite
+      view = ContentView.new(:name => "new content view", :organization_id => @organization.id)
+      [false, true].each do |composite|
```

Comment:
Why test non-composite here? I'd opt for testing only composite, since we test normal CV creating on L80 `test_create_with_name`.

Reply:
@akofink  we need to trigger both property values for this test, we have to be sure that we are able to create with with true false values.

Reply:
@akofink this test test cost nothing in terms of time, as it does create a real entity, but informative on the reason of failure

Reply:
> this test test cost nothing in terms of time
All the tests have some time cost associated, even if it doesn't hit the DB. Ensuring we don't test the same thing twice isn't pertinent, but it's ideal. Since L80 is testing `composite: nil` (i.e. default) instead of `composite: false`, I guess this is fine.